server/src/main/scala/services/graphstages/ArrivalSplitsFromAllSourcesGraphStage.scala

```diff
@@ -15,8 +15,7 @@ import services._
 import scala.collection.immutable.Map
 import scala.language.postfixOps
 
-
-case class UpdatedFlights(flights: Map[Int, ApiFlightWithSplits], updatesCount: Int, additionsCount: Int)
+case class UpdatedFlights_(flights: Map[Int, ApiFlightWithSplits], updatesCount: Int, additionsCount: Int)
```

Comment:
is this temporary? Could it have a better name?

Reply:
It's now only used in the legacy split stage, so can be dropped once we've switched to the new manifests. I'll rename it with `Legacy` rather than an underscore